server/helper.py

```diff
@@ -570,12 +570,17 @@ def resolve_device_name_pholus (pMAC, pIP, allRes, nameNotFound, match_IP = Fals
     
 
 #-------------------------------------------------------------------------------
+import dns.resolver
```

Comment:
if you need an additional python package you need to update the following files:

https://github.com/jokob-sk/NetAlertX/blob/ecf6c1848d8947d737ee826be1c99038a346ded0/install/install_dependencies.debian.sh#L33

https://github.com/jokob-sk/NetAlertX/blob/ecf6c1848d8947d737ee826be1c99038a346ded0/Dockerfile#L15